hapi-fees/src/main/java/com/hedera/services/usage/schedule/ScheduleOpsUsage.java

```diff
@@ -0,0 +1,118 @@
+package com.hedera.services.usage.schedule;
+
+/*-
+ * ‌
+ * Hedera Services API Fees
+ * ​
+ * Copyright (C) 2018 - 2021 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+import com.hedera.services.usage.EstimatorFactory;
+import com.hedera.services.usage.QueryUsage;
+import com.hedera.services.usage.SigUsage;
+import com.hedera.services.usage.TxnUsageEstimator;
+import com.hederahashgraph.api.proto.java.FeeData;
+import com.hederahashgraph.api.proto.java.Query;
+import com.hederahashgraph.api.proto.java.ResponseType;
+import com.hederahashgraph.api.proto.java.TransactionBody;
+
+import java.util.function.Function;
+
+import static com.hedera.services.usage.SingletonEstimatorUtils.ESTIMATOR_UTILS;
+import static com.hedera.services.usage.SingletonUsageProperties.USAGE_PROPERTIES;
+import static com.hedera.services.usage.schedule.entities.ScheduleEntitySizes.SCHEDULE_ENTITY_SIZES;
+import static com.hederahashgraph.fee.FeeBuilder.BASIC_ENTITY_ID_SIZE;
+import static com.hederahashgraph.fee.FeeBuilder.BASIC_RICH_INSTANT_SIZE;
+import static com.hederahashgraph.fee.FeeBuilder.BASIC_TX_ID_SIZE;
+import static com.hederahashgraph.fee.FeeBuilder.BOOL_SIZE;
+import static com.hederahashgraph.fee.FeeBuilder.getAccountKeyStorageSize;
+
+public class ScheduleOpsUsage {
+	/* Scheduled transaction ids have the scheduled=true flag set */
+	private static final long SCHEDULED_TXN_ID_SIZE = BASIC_TX_ID_SIZE + BOOL_SIZE;
+
+	static EstimatorFactory txnEstimateFactory = TxnUsageEstimator::new;
+	static Function<ResponseType, QueryUsage> queryEstimateFactory = QueryUsage::new;
+
+	public FeeData scheduleInfoUsage(Query scheduleInfo, ExtantScheduleContext ctx) {
+		var op = scheduleInfo.getScheduleGetInfo();
+
+		var estimate = queryEstimateFactory.apply(op.getHeader().getResponseType());
+		estimate.updateTb(BASIC_ENTITY_ID_SIZE);
+		estimate.updateRb(SCHEDULE_ENTITY_SIZES.fixedBytesInScheduleRepr() + ctx.nonBaseRb());
```

Comment:
I think that we shouldn't use `SCHEDULE_ENTITY_SIZES.fixedBytesInScheduleRepr()` since we are covering those properties in the `ctx.nonBaseRb` or am I mistaken?